src/psyclone/tests/psyir/nodes/omp_directives_test.py

```diff
@@ -621,8 +622,51 @@ class generates the expected code. Use the gocean API.
     clauses = ""
     if grainsize is not None:
         clauses = " grainsize({0})".format(grainsize)
+        if nogroup:
+            clauses = clauses + ", nogroup"
     if num_tasks is not None:
         clauses = " num_tasks({0})".format(num_tasks)
+        if nogroup:
```

Comment:
Can `nogroup` be `True` if grainsize *and* num_tasks are None? Might be nicer to specify the expected clauses string as part of the test parameterisation.

Reply:
To elaborate, I think I was worried that as written, this test wouldn't work correctly if the parameterisation was modified so that grainsize or num_tasks is None but `nogroup` is `True`. Then I thought it might be best to add `clauses` as one of the test parameters, e.g.:

    (None, None, False, ""), (32, None, True, "grainsize(32), nogroup"), ...

Reply:
Done